Sources/prefrence/ALUserDefaultsHandler.m

```diff
@@ -525,7 +525,7 @@ + (void)setUserAuthenticationTypeId:(short)type {
 + (short)getUserAuthenticationTypeId {
     NSUserDefaults *userDefaults = [ALUserDefaultsHandler getUserDefaults];
     short type = [userDefaults integerForKey:AL_USER_AUTHENTICATION_TYPE_ID];
-    return type ? type : 0;
+    return type ? type : 1;
```

Comment:
It would also affect existing users who were passing an empty password. See if it's fine, or we need to do something for those users.

Reply:
@mukeshthawani  No it will not affect existing user who were passing empty password. They can login with empty password.

Reply:
Verified it as well